src/components/StarRating/StarRating.ts

```diff
@@ -0,0 +1,63 @@
+import { $$, Utils, Component, Initialization, ComponentOptions, IComponentBindings } from 'coveo-search-ui';
+import { star as starIcon } from '../../utils/icons';
+
+export interface IStarRatingOptions {
+    /** Specifies the rating to be displayed as stars. */
+    rating: number;
```

Comment:
that should probably be a "field" instead of a number ? How will you get that number otherwise from the index ?

Reply:
What do you mean by "field"? The value is passed like in the example and number gets converted fine.